src/renderer/components/kube-object-status-icon/kube-object-status-icon.tsx

```diff
@@ -133,6 +139,10 @@ class NonInjectedKubeObjectStatusIcon extends React.Component<KubeObjectStatusIc
   }
 }
 
+function isNotEmpty<T>(item: T | null | undefined): item is T {
+  return !!item;
+}
```

Comment:
nit: this already exists in the form of `isDefined` in `src/common/utils/type-narrowing.ts`. Also this function is not quite correct since `isNotEmpty(0)` would return `false`

Reply:
Removed new implementation to prefer existing.